record-editor/src/app/shared/config/authors.config.ts

```diff
@@ -21,7 +21,7 @@
  */
 
 import { JsonEditorConfig } from 'ng2-json-editor';
-import { customValidationForDateTypes, anchorBuilder } from './commons';
+import { affiliationAutocompletionConfig, customValidationForDateTypes, anchorBuilder } from './commons';
```

Comment:
are you sure you have pre-commit hooks set up correctly? I would expect this to be sorted by `prettier`.

Reply:
Agree with Micha. Line 24 should have been automatically formatted to multiline import statement which is easier to read.